web/portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -205,6 +196,11 @@
   </filter-mapping>
 
   <filter-mapping>
+    <filter-name>LocalizationFilter</filter-name>
+    <url-pattern>/*</url-pattern>
```

Comment:
I agree there will not be a lot of additional processing, but since it is applied on all the requests, it could be impacting. Honestly I am quite confident, but as you said, it has to be validated by TQA.